docs/source/getting-started/installation.rst

```diff
@@ -300,28 +335,38 @@ increases before proceeding.
 Quick alternative: trying in Docker
 ------------------------------------------------------
 
-As a **quick alternative to installing SkyPilot on your laptop**, we also provide a Docker image with SkyPilot and
-its AWS and GCP dependencies installed for users to quickly try out SkyPilot. You can simply run:
+As a **quick alternative to installing SkyPilot on your laptop**, we also
+provide a Docker image with SkyPilot main branch automatically cloned.
+You can simply run:
 
 .. code-block:: console
 
-  $ docker run -td --name sky --rm -v "$HOME/.sky:/root/.sky:rw" -v "$HOME/.aws:/root/.aws:rw" -v "$HOME/.config/gcloud:/root/.config/gcloud:rw" berkeleyskypilot/skypilot:latest
+  $ # NOTE: '--platform linux/amd64' is needed for Apple silicon Macs
+  $ docker run --platform linux/amd64 \
+    -td --rm --name sky -e SKYPILOT_DEBUG=0 \
+    -v "$HOME/.sky:/root/.sky:rw" \
+    -v "$HOME/.aws:/root/.aws:rw" \
+    -v "$HOME/.config/gcloud:/root/.config/gcloud:rw" \
+    berkeleyskypilot/skypilot-debug
   $ docker exec -it sky /bin/bash
 
-If your cloud CLIs are already setup, your credentials will be mounted to the
-container and you can proceed to :ref:`Quickstart <quickstart>`. Else you can
-follow the instructions in :ref:`Cloud account setup <cloud-account-setup>`
-inside the container to setup your cloud accounts.
+  $ # Run inside the container:
 
-If you exit from the shell in the container, the container will keep running
-in the background. You can start a new shell with :code:`docker exec -it sky /bin/bash`.
+  $ cd /sky_repo/skypilot
```

Comment:
These lines are not needed with the nightly docker image

Reply:
Done.